src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -1135,5 +1234,99 @@ static void Foo(string str)
 {
 }", "param name=\"str\"");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ListTypes()
+        {
+            await VerifyItemsExistAsync(@"
+/// <summary>
+/// <list type=""$$""
+/// </summary>
+static void Foo()
+{
+}
+", "bullet", "number", "table");
+        }
+
+        [WorkItem(11490, "https://github.com/dotnet/roslyn/issues/11490")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task SeeLangwordValues()
+        {
+            await VerifyItemsExistAsync(@"
+/// <summary>
+/// <see langword=""$$""
+/// </summary>
+static void Foo()
+{
+}
+", "await", "class");
+        }
+
+        [WorkItem(11489, "https://github.com/dotnet/roslyn/issues/11489")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task SeeAttributesOnSpace()
+        {
+            var text = @"
+/// <summary>
+/// <see $$
+/// </summary>
+static void Foo()
+{
+}";
+            await VerifyItemExistsAsync(text, "langword", usePreviousCharAsTrigger: true);
```

Comment:
i see.  you don't want to insert ```<see cref``` because it might be ```<see langword```.  in this case, i think we should likely include all three items in the completion list.  That way people can still generate ```<see cref``` easily.  What do you think?  I'm open to not doing this as the new style may just work fine.